lib/pc_schema.c

```diff
@@ -335,28 +335,34 @@ void pc_schema_check_xyzm(PCSCHEMA *s)
 		char *dimname = s->dims[i]->name;
 		if ( ! dimname ) continue;
 		if ( strcasecmp(dimname, "X") == 0 ||
+			 strcasecmp(dimname, "transform.translation[0]") == 0 ||
```

Comment:
It is the x component of the translation part of an affine transform provided by a ROS [TransformStamped](http://docs.ros.org/jade/api/geometry_msgs/html/msg/TransformStamped.html) message through our [fdw-rosbag](LI3DS/fdw-rosbag). 

I agree that it is too specific and that ultimately, either : 
- the rosbag fdw expose the translation part using x,y,z column names
- or overriding the x,y,z,m dims should be made possible in the xml schema (preferred)